.github/workflows/codestyle.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           -   uses: actions/checkout@v3
           -   uses: shivammathur/setup-php@v2
               with:
-                  php-version: 8.1
+                  php-version: 8.2
```

Comment:
This is throwing the same error as #297 which might be an issue in the latest `friendsofphp/php-cs-fixer` though I cannot find an issue for it.

Do we need this package if we are using phpstan?

Reply:
Error has been found and fixed in main in related package, i'd probably opt for temporary dev-main for the dev package until the fix is published.